### PR TITLE
Tag QuickTypes.jl v0.1.3

### PR DESCRIPTION
**Pull request by attobot**
Repository: [cstjean/QuickTypes.jl](https://github.com/cstjean/QuickTypes.jl)
Release: [v0.1.3](https://github.com/cstjean/QuickTypes.jl/releases/tag/v0.1.3)
Travis: [Travis Build Status](https://travis-ci.org/cstjean/QuickTypes.jl/builds/219382168)
Diff: [vs v0.1.2](https://github.com/cstjean/QuickTypes.jl/compare/70b76dc5a6bca736dabdc2139c3a610ada99cea3...b492a6717626a42a561dc39032fcf85f31901316)
`requires` vs v0.1.2: no changes
cc: @cstjean

### CODE REVIEW

File: QuickTypes/versions/0.1.3/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.5
+MacroTools 0.3.2
+Compat 0.22.0
```
Comment (tkelman) [inline QuickTypes/versions/0.1.3/requires]: you aren't relying on Set conversions are you? Typeutils was 0.15.0

Comment (cstjean) [inline QuickTypes/versions/0.1.3/requires]: I'm using it for `roottype(typ::Type) =
VERSION < v"0.5.100" ? typ.name.primary : Compat.TypeUtils.typename(typ).wrapper`, to get `SomeType{T}` from `SomeType{Int}`. `roottype`isn't exported, but I rely on these functions in my other packages.

Comment (tkelman) [inline QuickTypes/versions/0.1.3/requires]: why is 0.22.0 the minimum version then?

Comment (cstjean) [inline QuickTypes/versions/0.1.3/requires]: I didn't know when TypeUtils.typename was defined. I'll change it...

**Pull request merged**